plugin/documents.py

```diff
@@ -483,8 +483,13 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
             scope = 'region.yellowish lightbulb.lsp'
             icon = 'Packages/LSP/icons/lightbulb.png'
         else:  # 'annotation'
-            suffix = 's' if action_count > 1 else ''
-            code_actions_link = make_command_link('lsp_code_actions', '{} code action{}'.format(action_count, suffix))
+            if action_count > 1:
+                title = '{} code actions'.format(action_count)
+            else:
+                title = next(iter(responses.values()))['title']
```

Comment:
This is unfortunately invalid code.

Code checker also says so. I recommend installing LSP-pyright which should notify you about most issues that the github checks does.

Reply:
Seems to work fine in the console:

```
>>> foo={"a":"b"}
>>> next(iter(foo.values()))
'b'
```

Reply:
I just installed `LSP-pyright`  

![image](https://user-images.githubusercontent.com/49985266/119616878-b9845880-be33-11eb-903b-12c513708786.png)

Before that, it was `pyls`

Reply:
Ah, I see my error now

```
>>> foo={"a":{"b":{"title": "hello there"}}}
>>> next(iter(foo.values()))["title"]
Traceback (most recent call last):
  File "__main__", line 1, in <module>
KeyError: 'title'
```

Reply:
I think you can use `next(iter(responses.values())[0]['title']` then. There must be a way to get the title in one line :)

Reply:
> I think you can use `next(iter(responses.values())[0]['title']` then. There must be a way to get the title in one line :)

Nice, it is very good